scripts/pree2e.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
 # install selenium
-../node_modules/selenium-standalone/bin/selenium-standalone install
+selenium-standalone install
 
-# start the app server
-npm start &
+# Creating Admin Test User
+{ echo admin@test.com; echo test; echo test; echo Admin Test User; echo admin;} | dotenv ./bin/cli-users create
```

Comment:
This isn't maintainable like this. We'll be adding the setup screen soon which will provide a GUI for creating the first admin user. This is too fragile.